packages/world/src/systems/ChipSystem.sol

```diff
@@ -84,24 +85,26 @@ contract ChipSystem is System {
 
     addToInventoryCount(playerEntityId, PlayerObjectID, ChipObjectID, 1);
 
-    Chip._setChipAddress(baseEntityId, address(0));
+    Chip._deleteRecord(baseEntityId);
+
+    address chipAddress = baseEntityId.getChipAddress();
```

Comment:
if you get chipAddress after the deleteRecord, wont it just be empty?

Reply:
thanks, we need tests lol